test/reactionsystem_core/coupled_equation_crn_systems.jl

```diff
@@ -167,7 +167,7 @@ let
     u0 = [X => 0.1, A => 16.1/2]
     nlprob = NonlinearProblem(coupled_rs, u0, ps, structural_simplify = true)
     nlsol = solve(nlprob)
-    @test nlsol ≈ [2.0, 3.0]
```

Comment:
because `structural_simplify` sometimes removes one of the variables now.

Reply:
This is better anyways -- we shouldn't rely on ordering of `us` or even what remains a `u`.